static/main.css

```diff
@@ -150,3 +157,64 @@ main.container {
 .ui-autocomplete { 
   z-index:2000; 
 }
+
+.user-events a{
+  color: #ff552e;
+  font-family: proxima-nova, sans-serif;
+  font-size: 20px;
+  font-weight: 800;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.2;
+  letter-spacing: normal;
+}
+
+.Events-Manager {
```

Comment:
Let's use `hyphenated-words-in-small-caps` convention for naming CSS classes.

Reply:
shall we create another issue to rename them? We can merge this one and @clav1d prs.